project/authentication/models.py

```diff
@@ -2,7 +2,222 @@
 """
 Copyright (c) 2019 - present AppSeed.us
 """
+import enum
+from datetime import timezone
 
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.contrib.auth.models import BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
 from django.db import models
 
+
 # Create your domain models here.
+
+
+class RecordStatus(enum.Enum):
+    """
+    This enum class represents the status of every record in the database.
+    The default record status for new database entries is ACTIVE.
+    """
+    ACTIVE = "Active"
+    DELETED = "Delete"
+    ACTIVE_LOCKED = "Active Locked"
+
+    def __str__(self):
+        return self.name
+
+
+class Gender(enum.Enum):
```

Comment:
The complexity thats currently roaming around regarding the values Gender can hold cannot be taken lightly. Lets shift this to Lookups and Lookup classes. The Lookup classes can be enums for now but we allow users to define the lookup values. This will also save us alot of time (short cut to avoid many system settings) by transferring the responsibility to the user to define the values they want. 
@umarkayondo  and @vgkienzler

Reply:
@deokyeyune @umarkayondo I added/changed a few things. See my [next commit ](https://github.com/vgkienzler/tickets-management-web/pull/11/commits/2dc013a6437450361f224e4c8f48b8e9d18c852b) in this branch.